thirdparty/gtest/CMakeLists.txt

```diff
@@ -33,7 +33,7 @@ foreach(target IN LISTS gtest_targets)
                 set_target_properties(${target} PROPERTIES COMPILE_OPTIONS "${_target_cxx_flags}")
             endif()
         endif()
-    elseif(CMAKE_COMPILER_IS_GNUCXX OR OV_COMPILER_IS_CLANG OR
+    elseif(CMAKE_COMPILER_IS_GNUCXX OR CMAKE_CXX_COMPILER_ID MATCHES "^(Apple)?Clang$" OR
```

Comment:
what's about `ov_disable_all_warnings` below?

Reply:
It is not disable `-Wno-undef`. Moreover if we use this CMake with PKG config we don't have set `OV_COMPILER_IS_CLANG` variable.